nexus/repositories.go

```diff
@@ -76,15 +76,15 @@ const (
 	RepositoryTypeGroup RepositoryType = "group"
 )
 
-func filterRepositoryJSONByFormat(jsonRepos json.RawMessage, format string) ([]byte, error) {
+func filterRepositoryJSONByFormat(jsonRepos json.RawMessage, format string, repoType RepositoryType) ([]byte, error) {
```

Comment:
Food for thought: right now it also filters by repository type, not only by format. `filterRepositoryJSONByFormatAndType` is **way** too verbose, but if we continue to add functionality to this we might want to consider the creation of a `RepositoryFilter` type and do use it like `repoFilter.ByFormat(format).ByType(type).BySomeOtherParameter(param).Filter()`.

I'm ok with how this is now, in any case. :-)

Reply:
I forget to change the signature name, I have to stop working on weekends. I'll change it to just `filterRepository`. The attributes give the context, we don't need to add a fluent structure to it nor add more verbosity.